app/views/planning_applications/neighbour_letter_batches/index.html.erb

```diff
@@ -14,30 +14,46 @@
 
 <h3 class="govuk-heading-s">Letters sent to neighbours</h3>
 
-<%= govuk_link_to "Download CSV", planning_application_consultation_neighbour_letter_batches_path(@planning_application, format: "csv") %>
+<p class="govuk-body govuk-!-margin-bottom-6">
+  <%= govuk_link_to "Download all as CSV", planning_application_consultation_neighbour_letter_batches_path(@planning_application, format: "csv") %>
+</p>
 
 <% if @planning_application.consultation.neighbour_letter_batches.present? %>
-  <% @planning_application.consultation.neighbour_letter_batches.each_with_index do |batch, i| %>
-    <%= govuk_accordion do |accordion| %>
-      <% accordion.with_section(heading_text: "Neighbour letter #{i + 1}", summary_text: "Date sent: #{batch.created_at.to_date.to_fs}") do %>
-        <%= govuk_link_to "Download letter", planning_application_consultation_neighbour_letter_batch_path(@planning_application, batch, format: "pdf") %>
-        <%= govuk_table do |table|
-              table.with_head do |head|
-                head.with_row do |row|
-                  row.with_cell(text: "Address")
-                  row.with_cell(text: "Sent by")
+  <%= bops_task_accordion(id: "neighbour-letter-batches") do |accordion| %>
+    <% accordion.with_heading(text: "Neighbour letters") %>
+    <% @planning_application.consultation.neighbour_letter_batches.each_with_index do |batch, i| %>
+      <% accordion.with_section do |section| %>
+        <% section.with_heading(text: "Neighbour letter #{i + 1}") %>
```

Comment:
I wonder if this is something we could add to the accordion component - a running counter of section number, e.g.

``` erb
<% section.with_heading(text: "Neighbour letter %{counter}") %>
```

Obviously not as part of this PR, but it's likely others will need something like it.

Reply:
In this case it would better be implemented as a property of the letter because it's also used in the CSV.